src/stories/Logo.stories.ts

```diff
@@ -0,0 +1,50 @@
+import type { Meta, StoryObj } from "@storybook/svelte";
+
+import Logo, { placements } from "$lib/components/Logo";
+
+const meta = {
+  title: "Components/Logo",
+  component: Logo,
+  tags: ["autodocs"],
+  argTypes: {
+    placement: {
+      control: { type: "select" },
+      options: placements,
+    },
```

Comment:
Unfortunately the preview doesn't actually update when changing this prop. I recall we had similar issues with other components, but I didn't feel that getting this working was worthwhile. The individual stories below serve as good enough examples of each use-case.